backend/Dockerfile

```diff
@@ -13,6 +13,7 @@ COPY . src
 RUN <<EOT
     set -e
     sed -i 's/plone.autoinclude==1.0.1/plone.autoinclude==2.0.0/g' /app/constraints.txt
+    sed -i 's/plone.exportimport==1.0.0/plone.exportimport==1.1.0/g' /app/constraints.txt
```

Comment:
Is it possible to install using the uv lockfile instead of constraints.txt so that it isn't necessary to update package versions in 2 places?

Reply:
Not using the current base image from phone -- as we need to install it using uv pip install, but I'm considering alternatives to it.